docs/configuration/identity-providers/oidc.md

```diff
@@ -309,6 +310,39 @@ This controls the minimum length of the `nonce` and `state` parameters.
 make certain scenarios less secure. It is highly encouraged that if your OpenID Connect RP does not send these parameters
 or sends parameters with a lower length than the default that they implement a change rather than changing this value.
 
+### enforce_pkce
+<div markdown="1">
+type: string
+{: .label .label-config .label-purple } 
+default: public_only
```

Comment:
```suggestion
default: always
```

Reply:
See https://github.com/authelia/authelia/pull/2924#issuecomment-1056113203

Reply:
Also, comment for myself, that should have been `default: public_clients_only`.

Reply:
Oh Yep! I'll make this change quickly if it hasn't been done. Good spot!